src/SdkCommon/ClientRuntime/Tests/ClientRuntime.NetCore.Tests/TransientFaultHandling/MemoryTest/CRMemoryTests.cs

```diff
@@ -1,31 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-namespace CR.Azure.NetCore.Tests.MemoryTest
```

Comment:
Please do not delete existing test.

Reply:
The test was previously testing that handlers are not being leaked by the RetryDelegatingHandler.  Since the RetryDelegatingHandler is not registering handlers anymore the test seemed obsolete, thus the removal.

I will bring back the test and update it to test the new behavior.  Now that the RetryDelegatingHandler is not making use of the Retrying handler, the test will validate that no handlers are being registered.